shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-fixture/src/test/java/org/apache/shardingsphere/test/integration/junit/runner/ShardingSphereRunner.java

```diff
@@ -17,281 +17,24 @@
-import org.apache.shardingsphere.test.integration.cases.IntegrationTestCasesLoader;
-import org.apache.shardingsphere.test.integration.cases.assertion.IntegrationTestCase;
-import org.apache.shardingsphere.test.integration.cases.assertion.IntegrationTestCaseAssertion;
-import org.apache.shardingsphere.test.integration.common.SQLExecuteType;
 import org.apache.shardingsphere.test.integration.env.EnvironmentType;
 import org.apache.shardingsphere.test.integration.env.IntegrationTestEnvironment;
-import org.apache.shardingsphere.test.integration.junit.annotation.BeforeAllCases;
-import org.apache.shardingsphere.test.integration.junit.annotation.ParameterFilter;
-import org.apache.shardingsphere.test.integration.junit.annotation.TestCaseSpec;
-import org.apache.shardingsphere.test.integration.junit.compose.ContainerCompose;
-import org.apache.shardingsphere.test.integration.junit.param.ParameterizedArrayFactory;
-import org.apache.shardingsphere.test.integration.junit.param.TestCaseParameters;
-import org.apache.shardingsphere.test.integration.junit.param.model.AssertionParameterizedArray;
-import org.apache.shardingsphere.test.integration.junit.param.model.CaseParameterizedArray;
-import org.apache.shardingsphere.test.integration.junit.param.model.ParameterizedArray;
 import org.apache.shardingsphere.test.integration.junit.runner.parallel.ParallelRunnerScheduler;
 import org.apache.shardingsphere.test.integration.junit.runner.parallel.annotaion.ParallelRuntimeStrategy;
-import org.junit.runner.Runner;
-import org.junit.runners.Suite;
-import org.junit.runners.model.FrameworkMethod;
-import org.junit.runners.model.InitializationError;
-import org.junit.runners.model.Statement;
+import org.junit.runners.Parameterized;
 
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
-import java.util.function.Predicate;
-import java.util.stream.Collectors;
-
-public final class ShardingSphereRunner extends Suite {
-    
-    private final List<Runner> runners;
-    
-    @Getter
-    private final String caseName;
-    
-    private ContainerCompose compose;
-    
-    private final TestCaseBeanContext beanContext = new TestCaseBeanContext();
-    
-    public ShardingSphereRunner(final Class<?> klass) throws InitializationError {
-        super(klass, Collections.emptyList());
-        TestCaseSpec testCaseSpec = getTestClass().getAnnotation(TestCaseSpec.class);
-        caseName = Strings.isNullOrEmpty(testCaseSpec.name()) ? klass.getSimpleName() : testCaseSpec.name();
-        if (EnvironmentType.NATIVE == IntegrationTestEnvironment.getInstance().getEnvType()) {
-            ParallelRuntimeStrategy parallelRuntimeStrategy = getTestClass().getAnnotation(ParallelRuntimeStrategy.class);
+public final class ShardingSphereRunner extends Parameterized {
+    
+    // CHECKSTYLE:OFF
+    public ShardingSphereRunner(final Class<?> klass) throws Throwable {
```

Comment:
It is better to keep the variable name consist, `klass` should rename to `clazz`